examples/urlInterpolationDemo.html

```diff
@@ -75,28 +75,36 @@ <h2>Localization Demo</h2>
                 for (var i = 0; i < pairs.length; i++)
                 {
                     var currParam = pairs[i].split('=');
-                    var currLi = '';
+                    var currLiDt = '';
```

Comment:
These vars are artifacts from a time we had to support IE10.  Would you mind converting these loop vars to lets and hoisting currLiDd up here?